tests/System.Text.Primitives.Tests/Encoding/Utf8Utf16ConversionTests.cs

```diff
@@ -78,6 +78,127 @@ public void EnumerateAndEnsureCodePointsOfTheSameUtf8AndUtf16StringsAreTheSame(s
             }
         }
 
+        public static object[][] InvalidUtf8ToUtf16SequenceData = {
```

Comment:
Should we add this to set of Encode/Decode tests for all encodings? And are the test values specific to this one implementation?